server/gitlab/api.go

```diff
@@ -267,6 +267,92 @@ func (g *gitlab) GetProject(ctx context.Context, user *UserInfo, owner, repo str
 	return result, nil
 }
 
+func (g *gitlab) GetYourProjects(ctx context.Context, user *UserInfo) ([]*internGitlab.Project, error) {
+	client, err := g.gitlabConnect(*user.Token)
+	if err != nil {
+		return nil, err
+	}
+	owned := true
+
+	result, resp, err := client.Projects.ListProjects(
+		&internGitlab.ListProjectsOptions{
+			Owned: &owned,
+		},
+		internGitlab.WithContext(ctx),
+	)
+	if respErr := checkResponse(resp); respErr != nil {
+		return nil, respErr
+	}
+	if err != nil {
+		return nil, err
+	}
+
+	return result, nil
+}
+
+func (g *gitlab) GetLabels(ctx context.Context, user *UserInfo, pid string) ([]*internGitlab.Label, error) {
+	client, err := g.gitlabConnect(*user.Token)
+	if err != nil {
+		return nil, err
+	}
+	var projectID interface{} = pid
+	result, resp, err := client.Labels.ListLabels(
+		projectID,
+		&internGitlab.ListLabelsOptions{},
+		internGitlab.WithContext(ctx),
+	)
+	if respErr := checkResponse(resp); respErr != nil {
+		return nil, respErr
+	}
+	if err != nil {
+		return nil, err
+	}
+
+	return result, nil
+}
+
+func (g *gitlab) GetMilestones(ctx context.Context, user *UserInfo, pid string) ([]*internGitlab.Milestone, error) {
+	client, err := g.gitlabConnect(*user.Token)
+	if err != nil {
+		return nil, err
+	}
+	var projectID interface{} = pid
+	result, resp, err := client.Milestones.ListMilestones(
+		projectID,
+		&internGitlab.ListMilestonesOptions{},
+		internGitlab.WithContext(ctx),
+	)
+	if respErr := checkResponse(resp); respErr != nil {
+		return nil, respErr
+	}
+	if err != nil {
+		return nil, err
+	}
+
+	return result, nil
+}
+
+func (g *gitlab) GetAssignees(ctx context.Context, user *UserInfo, pid string) ([]*internGitlab.ProjectMember, error) {
+	client, err := g.gitlabConnect(*user.Token)
+	if err != nil {
+		return nil, err
+	}
+	var projectID interface{} = pid
+	result, resp, err := client.ProjectMembers.ListProjectMembers(
+		projectID,
+		&internGitlab.ListProjectMembersOptions{},
+		internGitlab.WithContext(ctx),
+	)
+	if respErr := checkResponse(resp); respErr != nil {
+		return nil, respErr
+	}
+	if err != nil {
```

Comment:
same as above, we should check the error just after the action is performed, any reason why you are doing this after another action in between?